app/docker/models/container.js

```diff
@@ -91,6 +91,14 @@ export function ContainerStatsViewModel(data) {
     this.CPUCores = data.cpu_stats.cpu_usage.percpu_usage.length;
   }
   this.Networks = _.values(data.networks);
+  if (data.blkio_stats !== undefined) {
+    //TODO: take care of multiple block devices
```

Comment:
Hi @DarkAEther regarding this todo, do you think multiple block devices could be added to this PR also?

Reply:
Hi @sbusso ! Thanks for taking the time to have a look at my PR. I'll work on adding it in.

Reply:
Hi again @sbusso ! I got started on this, but ran into an issue: The Major and Minor number associated with the Block devices are not available when the page initializes. This makes it very hard to associate the stats data with a chart.

Data available regarding the Mounts when the page and charts are initialized (note that multiple mounts on the same device will only show one block device in the stats):

![image](https://user-images.githubusercontent.com/30438425/100728975-36d39100-33ee-11eb-918b-4799cf012dab.png)

Data format of the I/O stats (a combination of major and minor numbers uniquely identified a block device):

![image](https://user-images.githubusercontent.com/30438425/100729176-74d0b500-33ee-11eb-9db9-8fe92c8170b1.png)

I convert the above I/O stats into the following format to make it easier to work with (the key is the major:minor) :

![image](https://user-images.githubusercontent.com/30438425/100729299-95007400-33ee-11eb-9a93-715245f37d23.png)

This results in 2 blank charts appearing when the first set of data appears, but the charts do not get initialised at that point. The charts can only get initialized in the `UpdateRepeater` where the data about the major, minor numbers are available. Making things worse, it appears that the charts are not available in the DOM for access until the function completes executing.

This means that there is a weird behavior of:
1. CPU, Memory, Network charts load up (no IO charts at this point)
2. CPU, Memory and Network charts show first data point (this is when the IO charts are created and placed in DOM)
3. CPU, Memory and Network charts show second data point (this is when the IO charts are actually initialized)
4. CPU, Memory and Network charts show third data point (this is when the IO charts show the axes)
5. CPU, Memory and Network charts show third data point (the IO charts show their first data point).

Any advice would be great. Do let me know if I could explain any part of this in a better way. Thanks!

Reply:
@DarkAEther sorry for letting this ticket inactive for long time. As it has been validated we will create a ticket to improve it and move this PR forward.